README.md

```diff
@@ -289,6 +289,13 @@ if __name__ == "__main__":
     server.run(port=8000)
 ```
 
+The `accelerator` variable can be set to `"auto"`, and LitServe will select the appropriate GPU or CPU, whichever is available.
```

Comment:
`auto` is already the default, we should not instruct users to add it explicitly

So, remove it from the code, and instead add a section (under and expanded advanced bullet) that says: `accelerator` is `auto` by default.

If you wish to perform explicit placement, then pass the device manually + example.